src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private int area;
+    private Color color;
+
+    public Figure(int area, Color color) {
+        this.area = area;
+        this.color = color;
+    }
+
+    public abstract void draw();
+
+    public int getArea() {
+        return area;
```

Comment:
this method should be custom for each figure, let's make abstract and override it in subclasses

Reply:
done